src/components/Button/Button.module.css

```diff
@@ -0,0 +1,27 @@
+.button__wrap{
+    display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)